chrome/locale/en-US/zotero/zotero.dtd

```diff
@@ -126,6 +127,7 @@
 <!ENTITY zotero.item.textTransform.titlecase			"Title Case">
 <!ENTITY zotero.item.textTransform.sentencecase			"Sentence case">
 <!ENTITY zotero.item.creatorTransform.nameSwap      	"Swap First/Last Names">
+<!ENTITY zotero.item.openURL							"Open in Browser">
```

Comment:
I think "Open in Browser" is a little awkward, among other things because for most people this will already be in Firefox, and it's weird to refer to Firefox as "Browser" from within Firefox.

We could probably just reuse the existing "View Online" string, though that would be a little weirder for the URL field, and seems a bit anachronistic in general. And conversely, "Open URL" would work fine for the URL field, but would be a little weirder for this. Unless we have something obviously better, maybe stick with "View Online" for now, since we already have it?